builtin/settingtypes.txt

```diff
@@ -573,6 +579,11 @@ shadow_soft_radius (Soft shadow radius) float 5.0 1.0 15.0
 #    Requires: enable_dynamic_shadows, opengl
 shadow_sky_body_orbit_tilt (Sky Body Orbit Tilt) float 0.0 -60.0 60.0
 
+#    Tint sunlight during sunrise/sunset.
+#
+#    Requires: enable_dynamic_shadows, opengl
+enable_sun_tint (Tinted sunlight) bool false
```

Comment:
Looks like this depends on volumetric light, right?
So it should be dependent on that, and perhaps be moved under that option...

Reply:
No it doesn't. In fact, tinted sunlight is always present for the volumetrics, which I think is pretty reasonable to be honest, there's no real reason to not have that.